index.html

```diff
@@ -372,7 +372,7 @@ <h4>No physical access to the XenServer host?</h4>
                     <p>We strongly suggest to download only the current release. This is only for historical purpose.</p>
                     <div>
                         <p>You can find all previous XCP-ng releases here:</p>
-                        <a href="https://updates.xcp-ng.org/isos/">https://updates.xcp-ng.org/isos/</a>
+                        <a href="https://mirrors.xcp-ng.org/isos?https=1">https://mirrors.xcp-ng.org</a>
```

Comment:
I usually leave a trailing `/` when the URL points at a directory, but it seems to work without.

Reply:
Let me ask @julien-f if he's aware of something in the URL format about this.